cellprofiler/tests/test_pipeline.py

```diff
@@ -1308,6 +1308,28 @@ def test_07_01_metadata(self):
         self.assertEquals(ipd.metadata["foo"], "Bar")
         self.assertEquals(ipd.metadata["baz"], "Blech")
         
+    def test_08_01_save_pipeline_notes(self):
+        fd = cStringIO.StringIO()
+        pipeline = cpp.Pipeline()
+        module = ATestModule()
+        module.module_num = 1
+        module.notes.append("Hello")
+        module.notes.append("World")
+        pipeline.add_module(module)
+        module = ATestModule()
+        module.module_num = 2
+        module.enabled = False
+        pipeline.add_module(module)
+        expected = """[   1] [ATestModule]
```

Comment:
Have you considered making this a fixture (i.e. a plaintext file that’s read by the test)?

It might be cleaner and prevent anyone from creating an odd spacing bug.